src/_guides/language/effective-dart/design.md

```diff
@@ -944,9 +975,10 @@ than the index of the last item.
 This is consistent with core libraries that do the same thing.
 
 {:.good-style}
+<?code-excerpt "misc/test/effective_dart_test.dart (param-range)" replace="/expect\(//g; /, \/\*\*\//  \/\//g; /\);//g"?>
 {% prettify dart %}
-[0, 1, 2, 3].sublist(1, 3) // [1, 2].
-'abcd'.substring(1, 3)     // "bc".
+[0, 1, 2, 3].sublist(1, 3)  // [1, 2]
+'abcd'.substring(1, 3)  // 'bc'
```

Comment:
This is the only snippet that is actually tested. The nice thing is that each entire line (including the expression value in comments), is excerpt from the test.